pennylane/debugging.py

```diff
@@ -186,6 +187,23 @@ def reset_active_dev(cls):
         """Reset the global active device variable to None."""
         cls.__active_dev = None
 
+    @classmethod
+    def _execute(cls, batch_tapes):
+        """Execute tape on the active device"""
+        dev = cls.get_active_device()
+
+        valid_batch, _ = (
+            (batch_tapes, None)
+            if not dev.wires
+            else qml.devices.preprocess.validate_device_wires(batch_tapes, wires=dev.wires)
+        )
+
+        program, new_config = dev.preprocess()
+        new_batch, fn = program(valid_batch)
+
+        # TODO: remove [0] index once compatible with transforms
```

Comment:
Could you give more context about this comment? 🤔

Reply:
Right now we are only performing single measurement (`qml.debugging.some_measurement()`) on a single tape that corresponds to the qnode, so the expected output is always length 1. 

In the future, if we support transforms that split our circuit into multiple tapes, or we support multiple measurements, than the output will have length > 1. In the future, we can return a tuple here or add additional logic to process the results. This comment is just to remind us so we don't lose some of the output in the future.

Reply:
Nice! Might be nice to add a bit more text to the comment in that case! 😁